package.json

```diff
@@ -18,7 +18,7 @@
     "express": "^4.17.1",
     "heapdump": "^0.3.15",
     "jsonwebtoken": "^8.5.1",
-    "mediasoup": "github:versatica/mediasoup#v3",
+    "mediasoup": "^3.6.32",
```

Comment:
I noticed the lockfile didn't change. I think you need to use `npm update` to make sure that gets updated as well.

That said, a MediaSoup update seems like a big deal and we should probably do that independent of any other changes. Particularly ones involving changing how connection/reconnection works.

Reply:
I haven't seen any issues with the update but I can move that to another PR to avoid confusion.

Reply:
https://github.com/mozilla/dialog/pull/13